Black-Market/Black-Market/Models/UserDataResponse.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  UserResponse.swift
+//  Black-Market
+//
+//  Created by Junior Sancho on 8/2/23.
+//
+
+import Foundation
+
+struct UserDataResponse: Codable {
+    let accessToken: String
+    let refreshToken: String
+    let user: UserModel
+    
+    
+    enum CodingKeys: String, CodingKey {
+        case accessToken = "access_token"
+        case refreshToken = "refresh_token"
+        case user
+    }
+}
+
+struct UserModel: Codable {
+    let id: Int
```

Comment:
should this be a String instead? just curious, not sure what is the setup in the BE but it is often more flexible handle a String

Reply:
the backend type is Int.